content/docs/user-guide/assets/asset-processor/skip-startup-scan.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: Skip Startup Scan
+title: Asset Processor Skip Startup Scan
+description: Skips startup checks for assets modified while Asset Processor was closed.
+weight: 700
+toc: true
+---
+
+Skipping Startup Scan improves O3DE Editor's launch time if no assets are modified while Asset Processor was closed. You can switch startup scan modes at any time but need to restart Asset Processor for the change to take effects. Asset Processor saves your preference between sessions.

```

Comment:
Maybe talk to the why, ie use this mode for code development workflows or use it only when you are sure you are not making asset changes or asset processing impacting changes. Its really an "expert" mode to aid code updating and debugging iteration times.